tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -139,7 +148,7 @@ Object.Pipeline {
 		index	1
 
 		Object.Widget.copier.1 {
-			stream_name	'SSP0 Playback'
+			stream_name	'$HEADSET_PLAYBACK_PCM_STREAM_NAME'
```

Comment:
A minor nitpick: Wouldn't it be cleaner to have one commit to introduce the stream name variable and another to change the name?